modules/call-dispatch/src/lib.rs

```diff
@@ -86,7 +82,11 @@ decl_module! {
 impl<T: Trait> MessageDispatch<T::MessageId> for Module<T> {
 	type Message = (SpecVersion, Weight, <T as Trait>::Call);
 
-	fn dispatch(bridge: InstanceId, id: T::MessageId, message: Self::Message) -> Weight {
+	fn dispatch_weight(message: &Self::Message) -> Weight {
+		message.1
+	}
+
+	fn dispatch(bridge: InstanceId, id: T::MessageId, message: Self::Message) {
```

Comment:
Why don't we return `Weight` any more? I think we should still report back that lower amount of weight was used.

Reply:
We do not have anything to spent remaining weight now. Or do you meant we should refund relayer in this case?

Reply:
Yeah, I meant refunding.

Reply:
Nvm, I've found answer in another comment